README.md

```diff
@@ -42,67 +42,72 @@ under a single directory, but it is not necessary. To create a directory to
 hold your scrimmage projects and clone this scrimmage repo, run the following
 commands:
 
-    $ mkdir -p ~/scrimmage && cd ~/scrimmage
-    $ git clone https://github.com/gtri/scrimmage.git
+    mkdir -p ~/scrimmage && cd ~/scrimmage
```

Comment:
Have you checked that these still show as code blocks? I remember having issues with the $ and indentation when editing a tutorial file

Reply:
Yeah if you go to this branch and look at the README they show as code blocks still. They just copy as actual runnable commands instead of ones you have to delete the $ from